reflection/reflection.fbs

```diff
@@ -26,6 +26,11 @@ enum BaseType : byte {
     Union
 }
 
+table Attribute {
+    name:string (required);
+    value:string (required);
```

Comment:
values are not required, for example "deprecated" is an attribute without a value.